insights/parsers/upstart.py

```diff
@@ -0,0 +1,117 @@
+"""
+UPstart - Command ``initctl --system list``
+===========================================
+
+Parser to parser the output of ``initctl --system list`` commands.
+"""
+
+from insights import parser, CommandParser
+from insights.parsers import SkipException
+from insights.specs import Specs
+
+
+@parser(Specs.initctl_lst)
+class UPstart(CommandParser):
+    """
+    Class to parse the output of initctl command. It allows a
+    system administrator to communicate and interact with the
+    Upstart init(8) daemon and list the services managed by
+    Upstart init.
+
+    Sample output::
+
+        rc stop/waiting
+        vmware-tools start/running
+        tty (/dev/tty3) start/running, process 9499
+        tty (/dev/tty2) start/running, process 9495
+        tty (/dev/tty1) start/running, process 9493
+        tty (/dev/tty6) start/running, process 9507
+        tty (/dev/tty5) start/running, process 9505
+        tty (/dev/tty4) start/running, process 9502
+        tty (/dev/ttyS0) start/running, process 9509
+        plymouth-shutdown stop/waiting
+        control-alt-delete stop/waiting
+        rcS-emergency stop/waiting
+        readahead-collector stop/waiting
+        kexec-disable stop/waiting
+        quit-plymouth stop/waiting
+        rcS stop/waiting
+        prefdm stop/waiting
+        init-system-dbus stop/waiting
+        ck-log-system-restart stop/waiting
+        readahead stop/waiting
+        ck-log-system-start stop/waiting
+        splash-manager stop/waiting
+        start-ttys stop/waiting
+        readahead-disable-services stop/waiting
+        ck-log-system-stop stop/waiting
+        rcS-sulogin stop/waiting
+        serial stop/waiting
+
+    Raises:
+        SkipException: When nothing need to parse.
+
+    Attributes:
+        upstart_managed: This returns daemon details.
+        daemon_status: This returns daemon status.
+        dev_status: This returns device status.
```

Comment:
Since they have individual docstring in their definition function, here the items in `Attributes` can be removed.

Reply:
Done

Reply:
Do you just remove the description after the ":"?

Reply:
Yes as the docs are individually documented in individual attributes it is removed as per your suggestion